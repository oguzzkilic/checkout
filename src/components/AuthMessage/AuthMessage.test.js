import React from 'react';
import Renderer from 'react-test-renderer';
import AuthMessage from './AuthMessage';

import 'jest-styled-components';

describe('AuthMessage Component Tests', () => {
  it('renders correctly', () => {
    const tree = Renderer
      .create(<AuthMessage>Message from world</AuthMessage>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
