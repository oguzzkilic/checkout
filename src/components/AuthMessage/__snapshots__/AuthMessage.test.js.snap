// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthMessage Component Tests renders correctly 1`] = `
.c0 {
  margin-top: 1em;
  margin-bottom: 2em;
  text-align: left;
  font-size: 12px;
  line-height: 1.6;
}

<div
  className="c0"
>
  Message from world
</div>
`;
